package.json

```
@@ -11,6 +11,7 @@
     "@types/jest": "24.0.17",
     "@types/node": "12.7.0",
     "@types/react": "16.8.24",
+    "@types/react-bootstrap": "^0.32.20",
```

Can we set this to a static version?
```suggestion
    "@types/react-bootstrap": "0.32.20",
```